### PR TITLE
make gas edit tooltips use Tooltip

### PR DESCRIPTION
**Pull request by brad-decker**
1. Removes the 'NotificationModal' component pictured in the "before" below. :tada:
2. Removes the "GAS_PRICE_INFO_MODAL" and "GAS_LIMIT_INFO_MODAL" modal keys
3. Renames keys in the locales objects to dissasociate them from the concept of a modal
4. Makes the info icon a tooltip instead of a click to open/dismiss modal.
5. Makes @rekmarks happy

<details>
<summary>Before</summary>

</details>

<details>
<summary>After</summary>

</details>

### CODE REVIEW

File: ui/app/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js [hunk 1]
```diff
@@ -182,19 +181,19 @@ export default class AdvancedGasInputs extends Component {
       <div className="advanced-gas-inputs__gas-edit-rows">
         { this.renderGasInput({
           label: this.context.t('gasPrice'),
+          title: this.context.t('gasPriceInfoTooltipContent'),
```
Comment (rekmarks) [inline ui/app/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js:184]: We should rename the `title` parameter to `tooltipTitle` for great clarity!

**Review by rekmarks — CHANGES_REQUESTED** [review]
Sir, I have a _nit_.

File: ui/app/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js (hunk 1 above)
Comment (rekmarks) [reply ui/app/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js:184]: This comment should've been made on the `renderGasInput` function declaration but I'm sure you get the idea 😅

Comment (brad-decker) [reply ui/app/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js:184]: You got it

Comment (brad-decker) [reply ui/app/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js]: fixed

**Review by rekmarks — APPROVED** [review]
Wonderful!

**Pull request merged**